openlcs/libs/common.py

```diff
@@ -357,5 +357,9 @@ def is_prod():
     principal_name = conf.get("service_principal_hostname")
     if principal_name is None:
         return False
```

Comment:
With the change on function name, it's weird to return `False`, `None` would be better. Others LGTM, please go ahead for the access testing.

Reply:
Updated, thanks